edge/edge-core/src/main/java/org/apache/servicecomb/edge/core/EdgeInvocation.java

```diff
@@ -59,6 +59,7 @@ public void init(String microserviceName, RoutingContext context, String path,
     this.responseEx = new VertxServerResponseToHttpServletResponse(context.response());
     this.httpServerFilters = httpServerFilters;
     requestEx.setAttribute(RestConst.REST_REQUEST, requestEx);
+    setAfterCreateInvocationHandler(invocation -> context.put(RestConst.REST_INVOCATION_CONTEXT, invocation));
```

Comment:
if just need to do something after createInvocation
just override and call super first is enough?

Reply:
New setting logic is in the overridden `createInvocation()` method, please review the new commit